src/pages/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -2439,12 +2523,22 @@ It returns the initial state of the current Agent.
 
 #### Specifics
 
-|                        |                                                                                        |
-| ---------------------- | -------------------------------------------------------------------------------------- |
-| **Action**             | `login`                                                                                |
-| **Required scopes**    | `chats--access:ro` `customers:ro` `multicast:ro` `agents--all:ro` `agents-bot--all:ro` |
-| **Web API equivalent** | -                                                                                      |
-| **Push message**       | -                                                                                      |
+|                        |                                                                                                                                                              |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Action**             | `login`                                                                                                                                                      |
+| **Required scopes**    | `chats--access:ro` **\*** and `customers:ro` **\*\*** and `multicast:ro` **\*\*\*** and `agents--all:ro`**\*\*\*\*** and `agents-bot--all:ro` **\*\*\*\*\*** |
```

Comment:
@bkubiak there was a 'because currently, we assume that the requester agents-bot--all:ro - to get pushes about bot updateswants all the pushes, but in fact they can just request only part of them using pushes param - perhaps we will change it in the future' part after those methods. I did not put it here. Please confirm if that was a good decision :)

Reply:
Yes, it definitely shouldn't be here - it is internal info for you :)